trnsysGUI/storageTank/widget.py

```diff
@@ -417,18 +417,16 @@ def _addChildContextMenuActions(self, contextMenu: _qtw.QMenu) -> None:
         exportDdckAction = contextMenu.addAction("Export ddck")
         exportDdckAction.triggered.connect(self.exportDck)
 
-    @_tp.no_type_check
-    # pylint: disable=inconsistent-return-statements
-    def mouseDoubleClickEvent(
-        self, event: _qtw.QGraphicsSceneMouseEvent, isTest=False
-    ) -> (_tp.Optional)[ConfigureStorageDialog]:
+    def mouseDoubleClickEvent(self, event: _qtw.QGraphicsSceneMouseEvent):
```

Comment:
Much better this way, thanks!

Reply:
Please add return type, too. (`-> None`, I guess?)